test/e2e/common/setup-kubestellar.sh

```diff
@@ -29,6 +29,14 @@ fi
 kflex init --create-kind $disable_chatty_status
 : Kubeflex kind cluster created.
 
+:
+: -------------------------------------------------------------------------
+: Install the post-create-hooks for ocm and kubstellar controller manager
+:
+kubectl apply -f ${SRC_DIR}/../../../config/postcreate-hooks/ocm.yaml
```

Comment:
BTW, this needs to become irrelevant too. Currently this line is using something built from an older version of this same git repo. Instead it should use the local copy of the repo, as is done in place of the other PostCreateHook.

Reply:
@MikeSpreitzer can you explain more precisely what do you mean by "local copy of the repo" for the ocm post-create-hook ?